worker-preview-audit.test.py

```
@@ -0,0 +1,65 @@
+AUDIT = ROOT / "worker-preview-audit"
+
+
+class WorkerPreviewAuditTest(unittest.TestCase):
+    def run_audit(self, files):
+        with tempfile.TemporaryDirectory() as temp:
+            root = Path(temp)
+            for name, content in files.items():
+                path = root / name
+                path.parent.mkdir(parents=True, exist_ok=True)
+                path.write_text(content)
+            result = subprocess.run([AUDIT, "--json", "--repo", root], text=True, capture_output=True)
```

Ruff (0.16.3)

[error] `subprocess` call: check for execution of untrusted input

(S603)